Sources/xcproj/AEXML+XcodeFormat.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+import AEXML
+
+extension AEXMLDocument {
+
+    var xmlXcodeFormat: String {
+        var xml = "<?xml version=\"\(options.documentHeader.version)\" encoding=\"\(options.documentHeader.encoding.uppercased())\"?>\n"
+        xml += root._xmlXcodeFormat + "\n"
+        return xml
+    }
+
+}
+
+let attributesOrder: [String: [String]] = [
```

Comment:
Lol, I wouldn't expect the elements to be written in a custom order. What do you think about adding some tests for the methods/variables in this file?

Reply:
Actually I think it makes sense as if they were not ordered it may lead to random changes. Not a huge fan of such hardcoding but couldn't find a better solution in this case. This also can be not complete list and something can pop up in future, but it works for oss projects used for tests.

I've added a test that validates that write without changes does not create any changes and added it to tests for oss projects, so it should cover all these changes with one test. What other tests would you add?

Reply:
It's ok! Unless Apple publishes somewhere the order that they follow for these attributes, I don't think it's possible to do it in any other way. With the tests that you added should be ok. If something pops up in the future we can review the order.